templates/ts-apollo-fullstack/server/src/schema/schema.graphql

```diff
@@ -7,23 +7,53 @@ type Comment {
   description: String
 
   """@manyToOne field: 'comments', key: 'noteId'"""
-  note: Note
+  note(filter: NoteFilterUniqueInput): Note
```

Comment:
Excited to see this!

Reply:
Ah that should not be there (can't filter on a 1:1 record) - it should only be on a list.